cmd/up/stignore.go

```diff
@@ -27,6 +27,7 @@ import (
 	initCMD "github.com/okteto/okteto/cmd/init"
 	"github.com/okteto/okteto/cmd/utils"
 	"github.com/okteto/okteto/pkg/config"
+	o_error "github.com/okteto/okteto/pkg/errors"
```

Comment:
```suggestion
	oktetoError "github.com/okteto/okteto/pkg/errors"
```

Convention for Go is to not have underscores in variable names.

Reply:
Ah, that's true 😅. Thank you!